src/formatters/searchfox.js

```diff
@@ -63,5 +77,9 @@ export const getSearchfoxLink = (app, metric) => {
       ? `https://searchfox.org/mozilla-central/search?q=${allLanguagePatterns}&regexp=true`
       : `https://searchfox.org/mozilla-mobile/search?q=${allLanguagePatterns}&path=${searchfoxMap[app]}&regexp=true`;
   }
-  return undefined;
+
+  return (
```

Comment:
Let's check to make sure there is an entry for this app in sourcegraph, just as we do above for searchfox.

There may be some cases where we don't have a code search service for a repository.